### PR TITLE
Auth screens the codeur

### PR DESCRIPTION
**Pull request by BenCodeurr**

### CODE REVIEW

**Review by bright2kwame — CHANGES_REQUESTED** [review]
lets give a more descriptive name here button1 and button2

File: clearpath/app/[dashboard]/preferences/page.css [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+  display: flex;
+  justify-content: start;
+  align-items: center;
+  color: black;
+  background-color: white;
+  border-radius: 12px;
+  padding: 10px 20px;
+}
+
+.button1 h2 {
```
Comment (bright2kwame) [inline clearpath/app/[dashboard]/preferences/page.css:107]: Let add a more descriptive name here

File: clearpath/app/[dashboard]/preferences/page.css [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+.email {
+  font-family: 'SF Pro';
+  font-size: 21px;
+  font-weight: 400;
+  line-height: 16px;
+  text-align: left;
+  color: #64748B;
+}
+
+.flexColumnImportant {
```
Comment (bright2kwame) [inline clearpath/app/[dashboard]/preferences/page.css:42]: Lets stick with one naming comvention flex-column-important

File: clearpath/components/component/terminal/dailymotivation.tsx [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+      const apiKey = process.env.NEXT_PUBLIC_API_NINJAS_KEY;
+      if (!apiKey) {
+        setError("API key is missing. Check your .env.local file.");
+        setIsLoading(false);
+        return;
+      }
+
+      try {
+        const response = await fetch(
+          "https://api.api-ninjas.com/v1/quotes?category=happiness",
```
Comment (bright2kwame) [inline clearpath/components/component/terminal/dailymotivation.tsx:26]: Is this the only place we need this ? if yes then fine but if we will need it else then lets refactore it

**Review by bright2kwame — CHANGES_REQUESTED** [review]

**Comment by Samsonroyal** [comment]
> lets give a more descriptive name here button1 and button2

@CodesByBaafogAyo please refactor this accordingly; the best approach would be to have one style for all buttons since they are similar. Unless if they are accomplishing different purposes then we can have them take on descriptive names

File: clearpath/components/component/terminal/dailymotivation.tsx (hunk 3 above)
Comment (Samsonroyal) [reply clearpath/components/component/terminal/dailymotivation.tsx:26]: Yes this is the only place where the quotes are used; but I had spoken with @boypaida12 On how to refactor this code to fit Next.JS conventions

He needs to create a fetch function that can call the Quotes API and make it globally accessible by putting it in the utils folder

File: clearpath/app/[dashboard]/preferences/page.css (hunk 1 above)
Comment (Samsonroyal) [reply clearpath/app/[dashboard]/preferences/page.css:107]: @CodesByBaafogAyo

File: clearpath/app/[dashboard]/preferences/page.css (hunk 2 above)
Comment (Samsonroyal) [reply clearpath/app/[dashboard]/preferences/page.css:42]: @CodesByBaafogAyo

**Review by Samsonroyal — APPROVED** [review]

**Comment by Samsonroyal** [comment]
All the latest changes are already merged into development; I will be closing this PR @BenCodeurr

subsequent changes will be pushed into Dev and not main directly.

**Pull request closed without merging**